src/dotnet/APIView/APIViewWeb/Models/RenderedCodeFile.cs

```diff
@@ -8,61 +8,27 @@ namespace APIViewWeb.Models
 {
     public class RenderedCodeFile
     {
-        private CodeLine[] _rendered;
-        private CodeLine[] _renderedReadOnly;
-        private CodeLine[] _renderedText;
-
         public RenderedCodeFile(CodeFile codeFile)
         {
             CodeFile = codeFile;
         }
 
         public CodeFile CodeFile { get; }
 
-        public CodeLine[] Render(bool showDocumentation)
+        public CodeLine[] Render()
         {
-            //Always render when documentation is requested to avoid cach thrashing
-            if (showDocumentation)
-            {
-                return CodeFileHtmlRenderer.Normal.Render(CodeFile, showDocumentation: true);
-            }
-
-            if (_rendered == null)
-            {
-                _rendered = CodeFileHtmlRenderer.Normal.Render(CodeFile);
-            }
-
-            return _rendered;
+            //Always render to avoid cach thrashing
+            return CodeFileHtmlRenderer.Normal.Render(CodeFile);
```

Comment:
I am not sure why we can't cache it since we always return documentation. Cache rendered object helps on perf side when review revision is accessed so frequently.